qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -242,6 +243,10 @@ def run(
                 executed across all experiments; e.g., if parameter_binds is a
                 length-n list, and there are m experiments, a total of m x n
                 experiments will be run (one for each experiment/bind pair).
+            use_measure_esp: Whether to use ESP (excited state promoted) readout for the final
```

Comment:
```suggestion
            use_measure_esp: Whether to use excited state promoted (ESP) readout for the final
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-ibmq-provider/pull/952/commits/e1f6a02f48f83e35e94e3b49616b87a2f4870381.